src/transformers/cache_utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging import version
 
 from .configuration_utils import PretrainedConfig
+from .pytorch_utils import is_torch_greater_or_equal_than_2_7
```

Comment:
Nit: let's import and use `is_torch_greater_or_equal("2.7")` instead of adding a new `is_torch_greater_or_equal_than_2_7` 🤗

Reply:
changed to `is_torch_greater_or_equal` utility.

And I also aligned the the behavior `is_torch_greater_or_equal` with `is_torch_greater_or_equal_than_x_x`. The difference happens when there are suffix after version number. Like version "2.7.0.dev20250306"
`is_torch_greater_or_equal("2.7")` will return `False`, but `is_torch_greater_or_equal_than_2_7` will return True.

So I change the `is_torch_greater_or_equal` from `version.parse(importlib.metadata.version("torch")) >= version.parse(library_version)` to `version.parse(version.parse(importlib.metadata.version("torch")).**base_version**) >= version.parse(library_version)` , from me, it's reasonable, since "2.7.0.dev20250306" is still "2.7.0". But it depends on your taste, pls let me know which you prefer? Thx.